subgraph.yaml

```diff
@@ -2,128 +2,222 @@ specVersion: 0.0.2
           handler: handleCollateralWithdrawn
-        - event: LoanCreated(indexed uint256,indexed address,uint256,uint256,uint256,uint256)
-          handler: handleLoanCreated
-        - event: SignerAdded(indexed address)
-          handler: handleSignerAdded
-        - event: SignerRemoved(indexed address)
-          handler: handleSignerRemoved
-      file: ./src/mappings/loansMappings.ts
+      file: ./src/mappings/loans/daiLoansMappings.ts
+  # USDCLoans
   - kind: ethereum/contract
-    name: LendingPoolInterface
+    name: USDCLoans
     network: ropsten
     source:
-      address: "0x90Bb28aA21EE1BC8ed4826c4d261C54d847F4C59"
-      abi: LendingPoolInterface
+      startBlock: 7900000
+      address: "0xF908F681D083a4F4266E61D2De8D48D6e6C19119"
+      abi: USDCLoans
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.3
       language: wasm/assemblyscript
       entities:
-        - TokenDeposited
-        - TokenWithdrawn
-        - TokenRepaid
-        - PaymentLiquidated
+        - Loan
+        - LoanTerm
+        - CollateralDeposit
+        - CollateralWithdraw
+        - LoanRepayment
+        - Liquidation
+        - Borrower
+        - EthTransaction
       abis:
-        - name: LendingPoolInterface
-          file: ./abis/LendingPoolInterface.json
+        - name: USDCLoans
+          file: ./abis/LoansInterface.json
       eventHandlers:
-        - event: TokenDeposited(indexed address,uint256)
-          handler: handleTokenDeposited
-        - event: TokenWithdrawn(indexed address,uint256)
-          handler: handleTokenWithdrawn
-        - event: TokenRepaid(indexed address,uint256)
-          handler: handleDaiRepaid
-        - event: PaymentLiquidated(indexed address,uint256)
-          handler: handlePaymentLiquidated
-      file: ./src/mappings/lendingPools/daiLendingPoolMappings.ts
+        - event: LoanTermsSet(indexed uint256,indexed address,indexed address,uint256,uint256,uint256,uint256,uint256)
+          handler: handleLoanTermsSet
+        - event: LoanTakenOut(indexed uint256,indexed address,uint256)
+          handler: handleLoanTakenOut
+        - event: LoanRepaid(indexed uint256,indexed address,uint256,address,uint256)
+          handler: handleLoanRepaid
+        - event: LoanLiquidated(indexed uint256,indexed address,address,uint256,uint256)
+          handler: handleLoanLiquidated
+        - event: CollateralDeposited(indexed uint256,indexed address,uint256)
+          handler: handleCollateralDeposited
+        - event: CollateralWithdrawn(indexed uint256,indexed address,uint256)
+          handler: handleCollateralWithdrawn
+      file: ./src/mappings/loans/usdcLoansMappings.ts
+  # DAIInterestConsensus
   - kind: ethereum/contract
-    name: InterestConsensusInterface
+    name: DAIInterestConsensus
     network: ropsten
     source:
-      address: "0x226c667fac6413a1Df61d34Ec3A67c0DC8370D64"
-      abi: InterestConsensusInterface
+      startBlock: 7900000
```

Comment:
How do you calculate this? or it's based some blocks before you deploy the contracts??

Reply:
It is based some blocks before I deploy the contracts (first one).